### ISSUE TITLE
Accessibility: Opening a drawer or other window doesn't read the screen that was opened.

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#1754]

**Comment by Hixie** [comment]
This is on our short-term radar.
